CommandScreen.c

```diff
@@ -0,0 +1,83 @@
+#include "CommandScreen.h"
+
+#include "config.h"
+#include "CRT.h"
+#include "IncSet.h"
+#include "ListItem.h"
+#include "Platform.h"
+#include "StringUtils.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+/*{
```

Comment:
Please drop this part; auto-generated headers have been removed.

Reply:
done